### PR TITLE
Fix trivial issues as of a577c57bfa205a8e1f8a99bb1c350ba8b9e0b756

### PR DESCRIPTION
**Pull request by native-api**
Subj, see commit messages & review

### CODE REVIEW

File: docker/Dockerfile_i686 [hunk 1]
```diff
@@ -13,26 +13,25 @@ RUN curl -O -L https://download.qt.io/archive/qt/4.8/4.8.7/qt-everywhere-opensou
...
 
-RUN curl -O -L  https://www.cpan.org/src/5.0/perl-5.10.1.tar.gz  && \
+RUN	yum install autoconf automake bzip2 git freetype-devel gcc gcc-c++ libtool make pkgconfig -y && \
+    curl -O -L https://www.cpan.org/src/5.0/perl-5.10.1.tar.gz && \
 	tar -xzf perl-5.10.1.tar.gz && \
-	yum install autoconf automake bzip2 git freetype-devel gcc gcc-c++ libtool make pkgconfig  -y  && \
-	cd perl-5.10.1  && \
-	./Configure -des -Dprefix=$HOME/ffmpeg_build && \
+	cd perl-5.10.1 && \
+	./Configure -des -Dprefix="$HOME/ffmpeg_build" && \
```
Comment (abhiTronix) [inline docker/Dockerfile_i686:30]: @native-api **Please try this fix for x86:** `./Configure -des -Dccflags=-m32 -Dldflags=-m32 -Dlddlflags=-m32 -Dcc=gcc -Dprefix=$HOME/ffmpeg_build`

for forcing 32-bit mode

File: docker/Dockerfile_x86_64 [hunk 2]
```diff
@@ -13,26 +13,25 @@ RUN curl -O -L https://download.qt.io/archive/qt/4.8/4.8.7/qt-everywhere-opensou
...
 
-RUN curl -O -L  https://www.cpan.org/src/5.0/perl-5.10.1.tar.gz  && \
+RUN	yum install autoconf automake bzip2 git freetype-devel gcc gcc-c++ libtool make pkgconfig -y && \
+    curl -O -L https://www.cpan.org/src/5.0/perl-5.10.1.tar.gz && \
 	tar -xzf perl-5.10.1.tar.gz && \
-	yum install autoconf automake bzip2 git freetype-devel gcc gcc-c++ libtool make pkgconfig  -y  && \
-	cd perl-5.10.1  && \
-	./Configure -des -Dprefix=$HOME/ffmpeg_build && \
+	cd perl-5.10.1 && \
+	./Configure -des -Dprefix="$HOME/ffmpeg_build" && \
```
Comment (abhiTronix) [inline docker/Dockerfile_x86_64:30]: @native-api **Please try this fix for x64:** `./Configure -des -Dccflags=-m64 -Dldflags=-m64 -Dlddlflags=-m64 -Dcc=gcc -Dprefix=$HOME/ffmpeg_build`

for forcing 64-bit mode.

**Comment by native-api** [comment]
This is a PR against your PR. If you merge it, the commits will appear in your PR.

**Review by abhiTronix — COMMENTED** [review]
> This is a PR against your PR. If you merge it, the commits will appear in your PR.

@native-api Yes, I know. These changes will be final, but we need to fix [x64 and x86 arch Perl bug](https://github.com/skvark/opencv-python/pull/229#discussion_r310386071) that you discovered recently.  Please try changes I suggested below, I think these changes will fix it.

**Comment by native-api** [comment]
> These changes will be final

Not at all. They'll be just merged into your PR branch, you can add more commits after that.
You'll just need to `git pull` the changes to your local repo so that your local branch and Github's one are in sync again.

File: docker/Dockerfile_i686 (hunk 1 above)
Comment (native-api) [reply docker/Dockerfile_i686:30]: Getting link errors for `attrs.c` and `Zlib.c` in particular.

Comment (native-api) [reply docker/Dockerfile_i686:30]: https://arsv.github.io/perl-cross/ might help

**Comment by abhiTronix** [comment]
> Not at all. They'll be just merged into your PR branch, you can add more commits after that.
> You'll just need to git pull the changes to your local repo so that your local branch and Github's one are in sync again.

@native-api You getting this wrong. I meant as you already reformatted the docker files with proper format. I think I will make this PR as the final commit for my PR. But since there is another bug to solve that you mentioned and your assistance is required. Therefore I was asking you to test these suggested changes at your end. Thanks anyways

File: docker/Dockerfile_i686 (hunk 1 above)
Comment (abhiTronix) [reply docker/Dockerfile_i686:30]: ok thanks for the heads-up

**Pull request merged**